src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-topbar/scale-switcher/ScaleSwitcher.scss

```diff
@@ -5,20 +5,12 @@
 }
```

Comment:
We probably want to rename this component entirely — it's no longer a ScaleSwitcher; and it also doesn't have a "selected node" anymore. Let's ask what people think it should be called?

Reply:
chosen name is entityTypeIndicator

Reply:
Decided to have **EntityTypeIndicator** during 29/7/2020 code-review